region/compressor.go

```diff
@@ -0,0 +1,106 @@
+// Copyright (C) 2020  The GoHBase Authors.  All rights reserved.
+// This file is part of GoHBase.
+// Use of this source code is governed by the Apache License 2.0
+// that can be found in the COPYING file.
+
+package region
+
+import (
+	"encoding/binary"
+	"fmt"
+	"net"
+
+	"github.com/tsuna/gohbase/compression"
+)
+
+type compressor struct {
+	compression.Codec
+}
+
+func min(x, y uint32) int {
+	if x > y {
```

Comment:
that's cute, implementing `min()` with a greater than check :)

Reply:
Brain works in mysterious ways :D